### ISSUE TITLE
Serialize application management operations

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-home#487]
This is mostly an edge case: sometime, users are inpatients and try to perform many tasks at the same time, like installing apps and updating stack. This could lead to strange behavior. I wonder if we should serialize the operations, only starting one when the previous is ended. Just asking. @poupotte ?

**Comment by poupotte** [comment]
Yes, it's already the case for user application. However the stack isn't in this serialization. We should add it.
